src/components/views/dialogs/devtools/SettingExplorer.tsx

```diff
@@ -298,7 +298,7 @@ const SettingsList: React.FC<ISettingsListProps> = ({ onBack, onView, onEdit })
                                     <code>{i}</code>
                                 </AccessibleButton>
                                 <AccessibleButton
-                                    alt={_t("devtools|edit_setting")}
+                                    title={_t("devtools|edit_setting")}
```

Comment:
There are a few of these alt/title changes - are they related?

Reply:
Yes, `alt` is not a valid property on a div.

Reply:
👍